### ISSUE TITLE
Enhancement: Add month and year drop down for date picker

### ISSUE THREAD

**Issue by thorsten19** [owncloud/calendar#372]
**Expected behaviour**
Date picker should include a drop down to easily select a different month and year for new calendar entries.
This could be done similar to how it was included in the contacts app (for selecting birthdays) in Owncloud v.8.2 (see screenshot).

The date picker should ideally also be used so on the left top you can easily navigate to a different month/year.

**Actual behaviour**
Selecting a different Month/year requires to click left/right. For calendar entries that vary from the current date more than just a few months this gets cumbersome.

**Server configuration**
Operating system: Raspbian GNU/Linux 8 (jessie)
Web server: Apache
Database: MySql
PHP version: 5.5.44
ownCloud version: (see ownCloud admin page) 9.0.0
Calendar 1.0

**Updated from an older ownCloud or fresh install:** Fresh install
**Client configuration**
Browser: Firefox 45.0.1
Operating system: Windows 7

![date-picker-oc82](https://cloud.githubusercontent.com/assets/18091239/14062560/071a2daa-f3a4-11e5-937b-f106143c7915.PNG)

**Comment by georgehrke** [comment]
Hm, i think the date picker in contacts and in calendar have different use cases. To set a birthday you almost always need to go back before the year 2000.
Going back before 01.01.2000 is something you very rarely do in the calendar app.

**Comment by georgehrke** [comment]
also cc @jancborchardt

**Comment by georgehrke** [comment]
concerning the sidebar:
The problem is, that the top bar of the date picker triggers its visibility. Not sure where to selects to select month and year.

**Comment by thorsten19** [comment]
Agree that the use case is different and going back in the calendar does not require more than couple of years.
Had a look at at the software I use at work (see screenshot) and the date picker in the calendar allows to go back and forward couple of months (sliding 1 year time window). In addition you can go back and forward couple of years. 
It would already be great if there is a smart way to quickly navigate between years, for example if "month" view is selected in the sidebar, the year could be a drop down. 

![datepicker-month](https://cloud.githubusercontent.com/assets/18091239/14066487/e5088ebc-f44a-11e5-86e5-b77f75471709.png)
![datepicker-year](https://cloud.githubusercontent.com/assets/18091239/14066488/e50b659c-f44a-11e5-8e76-f3d978a260e3.png)

**Comment by georgehrke** [comment]
I can totally see your need for such a feature, using a right-click action is not something we generally do in ownCloud.
I could imagine adding a button for something like "Go to ...". Probably in the settings as this is used very rarely. cc @jancborchardt 

<img width="551" alt="calendar - owncloud chromium today at 10 09 47 am" src="https://cloud.githubusercontent.com/assets/1250540/14101701/6b9fa608-f596-11e5-8a13-5a87096b1661.png">

**Comment by enoch85** [comment]
What, is there a datepicker in 1.1? :open_mouth: 

**Edit** Aah, now I see, you have to click it. Not there by default. Expected @jancborchardt ?

**Comment by jancborchardt** [comment]
Yeah, you have to click the date. No reason to show it all the time.
